### ISSUE TITLE
Tab complete should "just work" for builtin commands

### ISSUE THREAD

**Issue by bradleyjensen** [bradleyjensen/shcl#23]
Tab complete should know about `shell-lambda` options and flags and offer completions accordingly.

**Comment by qwertystop** [comment]
I am interested in this project and would like to help, but have effectively no experience with Common Lisp beyond working through two books (_Gentle Introduction to Symbolic Computation_ and _Practical Common Lisp_). This issue seems (from a naive view) to potentially be a good place to start (fairly well-defined and of limited scope). Would you agree with that, or would you recommend I start on a different issue?

**Comment by dbjergaard** [comment]
@qwertystop I agree, from a programming point of view this should be straightforward.  It might be hard to figure out where the appropriate hooks are in the codebase, so chime in here if you need help!
